### PR TITLE
made card

### PR DESCRIPTION
**Pull request by sobiiya**

### CODE REVIEW

**Review by Tauqeer-Ahmed-99 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                <div class="content">
+                    <img src="data.png" alt="">
+                    <h3>Code</h3>
+
+                </div>
+            </div>
+            <div class="face face2">
+                <div class="content">
+                    <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Deleniti sint ipsam similique aperiam consequatur id, tempora aspernatur vero fugiat molestiae repellat repudiandae accusamus</p>
+                    <a href="">Read More</a>
```
Comment (faran4engg) [inline index.html:44]: Not to break the app, add # to href

File: style.css [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        justify-content: center;
+        align-items: center;
+        position: relative;
+        display: flex;
+        background: #333333;
+        z-index: 1;
+        transform: translateY(100px);
+    }
+    
+    .container .card:hover .face.face1 {
```
Comment (faran4engg) [inline style.css:41]: Again, this, hover thing can be improved

File: style.css [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+        justify-content: center;
+        align-items: center;
+        position: relative;
+        display: flex;
+        background: #ffffff;
+        padding: 20px;
+        box-sizing: border-box;
+        box-shadow: 0 20px 50px rgba(0, 0, 0, 0.8);
+        transform: translateY(-100px);
+    }
```
Comment (faran4engg) [inline style.css:77]: Group the properties for better readability

File: index.html (hunk 1 above)
Comment (sobiiya) [reply index.html:44]: okay

File: style.css [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+    
+    .container .card .face.face2 {
+        background: #ffffff;
+        padding: 20px;
+        box-sizing: border-box;
+        box-shadow: 0 20px 50px rgba(0, 0, 0, 0.8);
+        transform: translateY(-100px);
+    }
+    
+    .container .card:hover>.face.face1 {
```
Comment (faran4engg) [inline style.css:52]: I didn't get the purpose of this :)

File: style.css [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+        background-color: #ff0057;
+    }
+    
+    .container .card:hover>.face.face1,
+    .container .card:hover>.face.face2 {
```
Comment (faran4engg) [inline style.css:57]: same here !

File: style.css (hunk 4 above)
Comment (sobiiya) [reply style.css:52]: my face1 div is directly inside my card container that's why I used >

Comment (faran4engg) [reply style.css:52]: then only the `.container .card:hover` should be enough ! why to add classes of child elements !

File: style.css [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+        box-sizing: border-box;
+        box-shadow: 0 20px 50px rgba(0, 0, 0, 0.8);
+        transform: translateY(-100px);
+    }
+    
+    .container .card:hover>.face.face1 {
+        background-color: #ff0057;
+    }
+    
+    .container .card:hover>.face.face1,
```
Comment (sobiiya) [inline style.css:56]: and here I combine similar property

File: style.css (hunk 4 above)
Comment (sobiiya) [reply style.css:52]: bacause on card  hover I want to chnage my child class property

Comment (faran4engg) [reply style.css:52]: ok, good job with the explanation! approving...

**Review by faran4engg — APPROVED** [review]

File: style.css (hunk 4 above)
Comment (sobiiya) [reply style.css:52]: Thank you :)

**Comment by sobiiya** [comment]
@faran4engg  should I merge pull request?

**Comment by faran4engg** [comment]
> @faran4engg should I merge pull request?

Yes.

**Pull request merged**